src/app/pages/test-report-dashboard/components/review-feedback/review-feedback.ts

```diff
@@ -43,11 +43,17 @@ export class ReviewFeedback {
 
   characterCountChanged(charactersRemaining: number) {
     this.feedbackCharsRemaining = charactersRemaining;
+    if (charactersRemaining <= -1) {
+      this.form.controls.feedback.patchValue(this.form.controls.feedback.value.slice(0, -1));
+    }
   }
 
   getCharacterCountText() {
     const characterString = Math.abs(this.feedbackCharsRemaining) === 1 ? 'character' : 'characters';
-    const endString = this.feedbackCharsRemaining < 0 ? 'too many' : 'remaining';
+    const endString = 'remaining';
+    if (this.feedbackCharsRemaining <= 0) {
+      return 'You have 0 characters remaining';
+    }
```

Comment:
I would imagine this should be redundant with a hard limit, the return value would be 

```
return `You have ${Math.abs(this.feedbackCharsRemaining)} ${characterString} remaining`;
```